Packs/Securonix/Integrations/Securonix/Securonix.py

```diff
@@ -464,6 +495,21 @@ def extract_closing_comments(activity_data: List[Dict[str, Any]], close_states_o
     return " | ".join(closing_comments)
 
 
+def escape_spotter_query(original_query: str) -> str:
```

Comment:
Please add unit test

Reply:
This function is covered as a part of the command unit test cases where we're providing query which will eventually fall to this function and validates it's behavior.